doc/nrf/libraries/zigbee/lib_zigbee_app_helper_functions.rst

```diff
@@ -0,0 +1,322 @@
+﻿.. _lib_zigbee_app_helper_functions:
+
+Zigbee application helper functions
```

Comment:
I think we should remove "Zigbee application helper functions" and just discuss the signal handler here.